src/validator/FetchEdgesValidator.cpp

```diff
@@ -284,5 +288,30 @@ std::string FetchEdgesValidator::buildRuntimeInput() {
     return inputVar_;
 }
 
+Expression* FetchEdgesValidator::emptyEdgeKeyFilter(const PlanNode *input) {
+    // _src != empty && _dst != empty && _rank != empty
+    DCHECK_GE(geColNames_.size(), 3);
+    auto *colEmptyExpr1 = new RelationalExpression(
```

Comment:
`colEmptyExpr1` modify to `srcIdEmptyExpr`

Reply:
Fixed.